docs/blog/newsletter-april-2024.rst

```diff
@@ -0,0 +1,143 @@
+
+Would metrics (other than readability) work with documentation? One person suggested relying on objective metrics such as lack of typos, adherence to style, and correctness. Others suggested focusing on plain, simple language or measuring how long it takes someone to solve the problem that brought them to the documentation.
+
+For information about readability, see `The Measurement of Readability <https://dl.acm.org/doi/pdf/10.1145/344599.344630>`__. If you've been asked to include readability scores and question their use, these resources may strengthen your position: `Readability formulas: 7 reasons to avoid them and what to do instead <https://www.uxmatters.com/mt/archives/2019/07/readability-formulas-7-reasons-to-avoid-them-and-what-to-do-instead.php>`__, `What makes writing more readable? <https://pudding.cool/2022/02/plain/>`__, and
+`Redish on readability formulas <https://redish.net/wp-content/uploads/Redish_on_Readability_Formulas.pdf>`__.
+
+-------------------------------------------
+Accelerating Documentation Creation with AI
+-------------------------------------------
+
+Documentarians weighing whether to use AI tools in their workflows have many concerns to address, such as content quality, data security, and ethical implications. In many fields, accuracy and reliability are paramount and can't be sacrified for speed.
+
+These topics were relevant to a recent thread in the `#ai channel <https://writethedocs.slack.com/archives/C1NEAD7D4>`__. The question was whether, as some `public commenters have claimed <https://twitter.com/samjulien/status/1765866411404976282>`__, AI can already speed up the creation of product documentation.
+
+Some respondents were adamant that it doesn't help them. Some because their company's security policies forbid its use. Others found it useful for generic content, where it can act like a focused web search but without any ads, but not for specialized questions about a product. Others said it wouldn't help them because writing the first draft is how they learn about the product.
+
+Some of the people who didn't find it personally helpful thought it might help non-writers or non-native speakers. People who struggle getting words on the page might benefit from the ability of large-language models (LLMs) to put together sentences in a commonly understood order.
+
+Others were adamant that LLMs have really helped them speed up their documentation process. For security, it's important to choose a service that doesn't share prompts and responses with the LLM providers for training. 
```

Comment:
Second use of adamant?

Reply:
It was supposed to be intention (some are adamant that X, others are adamant that Y). 🤔

Reply:
Hmmm, I think to me Adamant is a really specific level of belief, so I would not expect both sides to be equally specifically adamant. But I won't fight it, I'm not that adamant.

Reply:
From the conversation, it seemed like both sides were very strong in their beliefs. 🤷